src/Pages/Contact.tsx

```diff
@@ -41,43 +41,43 @@ const FlexSpacer = styled.div`
 export default function Contact() {
     const coordinators: CoordinatorCardProps[] = [
         {
-            name: "Konrad Maciejewski",
-            email: "konrad.maciejewski@hacknarok.pl",
+            name: "Klaudia Sulowska",
+            email: "@hacknarok.pl",
             position: "Koordynator ds. Promocji",
-            src: "img/coordinators/hack8/konrad.JPG",
-            placeholderSrc: "img/coordinators/julia_thumb_10.jpg",
+            src: "",
+            placeholderSrc: "",
             animationDelay: 0,
         },
         {
-            name: "Jakub Kłoczko",
-            email: "jakub.kloczko@hacknarok.pl",
+            name: "Michał Filipkowski",
+            email: "@hacknarok.pl",
             position: "Koordynator ds. Logistyki i Barteru",
-            src: "img/coordinators/hack8/jakub.JPG",
-            placeholderSrc: "img/coordinators/kacper_thumb_10.jpg",
+            src: "",
+            placeholderSrc: "",
             animationDelay: 100,
         },
         {
-            name: "Angelika Żyła",
-            email: "angelika.zyla@hacknarok.pl",
-            position: "Koordynatorka Główna",
-            src: "img/coordinators/hack8/angelika.JPG",
-            placeholderSrc: "img/coordinators/natalia_thumb_10.jpg",
+            name: "Adrian Jaśkowiec",
+            email: "adrian.jaskowiec@hacknarok.pl",
+            position: "Koordynator Główny",
+            src: "",

```

Comment:
Jeżeli brak zdjęcia nie rozjeżdża strony to spoko

Reply:
nawet nie